src/radha/Find2ndMax/Assignment30/Find2ndMaxFromArray.java

```diff
@@ -0,0 +1,32 @@
+/*
+Assignment - 30 : 4th May'2021
+
+WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55 
+ */
+package radha.Find2ndMax.Assignment30;
+
+public class Find2ndMaxFromArray {
+
+	int get2ndMaxNum(int[] arr) {
+		int maxNum;
+		for (int index = 0; index < arr.length; index++) {
+			for (int innerIndex = index + 1; innerIndex < arr.length; innerIndex++) {
```

Comment:
You can handle this with single loop as well.
![Uploading Screenshot 2021-05-04 at 9.12.19 PM.png…]()